src/itest/java/fi/helsinki/moodi/moodle/AbstractMoodleIntegrationTest.java

```diff
@@ -168,26 +192,30 @@ private Map curVariables(String courseId, List<StudentUser> students, List<Teach
     }
 
     private void expectTeachers(List<TeacherUser> teachers) {
-        mockSisuServer.expectPersonsRequest(teachers.stream().map(t -> t.personId).collect(Collectors.toList()),
+        mockSisuGraphQLServer.expectPersonsRequest(teachers.stream().map(t -> t.personId).collect(Collectors.toList()),
             "/sisu-itest/persons-itest.json",
             new ImmutableMap.Builder<String, List<TeacherUser>>()
                 .put("teachers", teachers).build());
     }
 
-    protected void expectCourseRealisationWithUsers(String courseId, List<StudentUser> students, List<TeacherUser> teachers) {
-        mockSisuServer.expectCourseUnitRealisationRequest(courseId,
-            "/sisu-itest/course-unit-realisation-itest.json",
+    protected void expectCourseRealisationsWithUsers(String courseId, List<StudentUser> students, List<TeacherUser> teachers) {
```

Comment:
Miksi nimi on muuttunut monikolliseksi vaikka paramiksi annetaan vain yksi kurssi-id?

Sama alemmassa `resetAndExpectCourseRealisationsWithUsers`-metodissa.

Reply:
Koska Sisuun päin menevä kutsu on monikollinen course_unit_realisations.